docker/pypi/wmagent/init.sh

```diff
@@ -313,7 +315,10 @@ check_databases() {
     esac
 
     # Checking CouchDB:
-    _check_couch
+    [[ -n $COUCH_HOST ]] && [[ -n $COUCH_PORT ]] && [[ -n $COUCH_USER ]] && [[ -n $COUCH_PASS ]] &&
+      couchdbCred=true
+    $couchdbCred || { echo "$FUNCNAME: ERROR: No couch credentials provided at $WMA_SECRETS_FILE" ; return $(false); }
+    _check_couch || return
```

Comment:
Dario, other than skipping the next line, was there any other reason to add this `return` statement here?

Reply:
I think the dashed lines in the echo below give us a nice separation in the logs. If it is not separated (with a dashed line) by a different method, I would be in favor of removing the return such that we execute the echo statement below.

Reply:
oook